src/client/app/containers/app.ts

```diff
@@ -73,12 +73,11 @@ export class AppComponent {
         return this.profile$
             .filter(profile => profile != null)
             .map(profile => {
-                if (!isNil(this.snippet.gistOwnerId)) {
-                    return this.snippet.gistOwnerId === profile.login;
+                if (!isNil(this.snippet.gist)) {
+                    return !isNil(this.snippet.gistOwnerId) ? this.snippet.gistOwnerId === profile.login : true;
```

Comment:
And also a comment about the "true", saying that we'll optimistically assume "true" for any snippet that doesn't specify an owner, for back-compat / lightup.